app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{#link-to "rentals.show" rentalUnit.slug}}
```

Comment:
Sorry for the delay in getting back to you on this.

The issue with turning the title into a link-to is that a `link-to`'s a tag is a block element whereas we need that h3 to be inline (otherwise it messes up all the styling).  It should only be 1 line of css in the add-on repo.  If we went with the nav option, we would have to add css anyway.

Alternatively, we could use an anchor and just add an inline style:
`<h3><a href="rental/show/{{rental.slug}}"style="display: inline">{{rental.title}}</a></h3>`